docs/rst/user_manual/configuration.rst

```diff
@@ -43,19 +50,66 @@ Each Topic is determined by its entries ``name``, ``type`` and ``keyed``, with o
 
     *   - ``keyed``
         - ``bool``
-        - ``false``
+        - Both ``true`` and ``false``
 
 The entry ``keyed`` determines whether the corresponding topic is `keyed <https://fast-dds.docs.eprosima.com/en/latest/fastdds/dds_layer/topic/typeSupport/typeSupport.html#data-types-with-a-key>`_
 or not. See :term:`Topic` section for further information about the topic.
 
-.. todo:
+.. note::
+
+    Tags ``allowlist``, ``blocklist`` and ``builtin-topics`` must be at yaml base level (it must not be inside any
+    other tag).
+
+.. note::
+
+    Placing quotation marks around values in a YAML file is generally optional. However, values containing wildcard
+    characters must be enclosed by single or double quotation marks.
+
+Allow topic list (``allowlist``)
+--------------------------------
+This is the list of topics that |ddsrouter| will forward, i.e. the data published under the topics matching the
+expressions in the ``allowlist`` will be relayed by |ddsrouter|.
+
+
+Block topic list (``blocklist``)
+--------------------------------
+This is the list of topics that the |ddsrouter| will block, that is, all data published under the topics matching the
+filters specified in the ``blocklist`` will be discarded by the |ddsrouter| and therefore will not be relayed.
+
+This list takes precedence over the ``allowlist``.
+If a topic matches an expression both in the ``allowlist`` and in the ``blocklist``, the ``blocklist`` takes precedence,
+causing the data under this topic to be discarded.
+
+
+Builtin topics list (``builtin-topics``)
+----------------------------------------
+
+|ddsrouter| includes a mechanism to automatically detect which topics are being used in a DDS network.
+By automatically detecting these topics, a |ddsrouter| creates internal DDS :term:`Writers<DataWriter>`
+and :term:`Readers<DataReader>` for each topic and for each Participant in order to relay the data published on each
+discovered topic.
```

Comment:
I would move this to the main Topic filtering paragraph, I think it is main information.
If not this, at least mention it above

Reply:
It is actually in the first paragraph.